build-common/NHibernate.props

```diff
@@ -21,6 +21,7 @@
     <NhNetCoreApp>false</NhNetCoreApp>
     <NhNetCoreApp Condition="$(TargetFramework.StartsWith('netcoreapp'))">true</NhNetCoreApp>
     <DefineConstants Condition="$(NhNetCoreApp) AND $(NhVbNet) == ''" >NETCOREAPP2_0_OR_GREATER;$(DefineConstants)</DefineConstants>
+    <WarningsNotAsErrors>NU1901;NU1902;NU1903;NU1904</WarningsNotAsErrors>
```

Comment:
We can no more compile the 5.3.x branch without downgrading from errors these warnings back to warnings.

Reply:
Just tested GHA build on my 5.3.x branch - it builds without any issues:
https://github.com/bahusoid/nhibernate-core/actions/runs/7003210288/job/19048468922